diff -rupN --no-dereference netgen-6.2.2406/CMakeLists.txt netgen-6.2.2406-new/CMakeLists.txt
--- netgen-6.2.2406/CMakeLists.txt	2024-12-03 18:26:11.119472907 +0100
+++ netgen-6.2.2406-new/CMakeLists.txt	2024-12-03 18:26:11.383465621 +0100
@@ -296,7 +296,7 @@
        target_include_directories(netgen_gui INTERFACE ${TK_INCLUDE_PATH}/../xlib)
     endif()
 
-    target_link_libraries(nggui PUBLIC nglib togl PRIVATE "$<BUILD_INTERFACE:netgen_python>" )
+    target_link_libraries(nggui PUBLIC nglib togl jpeg avcodec avformat avutil swscale PRIVATE "$<BUILD_INTERFACE:netgen_python>" )
 
     if(WIN32)
       target_compile_definitions(netgen_gui INTERFACE -DTOGL_WGL)
@@ -633,7 +633,7 @@
   file( WRITE "netgen_mesher-py3.egg-info"
 "Metadata-Version: 2.1
 Name: netgen-mesher
-Version: ${NETGEN_VERSION_MAJOR}.${NETGEN_VERSION_MINOR}.${NETGEN_VERSION_PATCH}.post${NETGEN_VERSION_TWEAK}
+Version: ${NETGEN_VERSION_MAJOR}.${NETGEN_VERSION_MINOR}.${NETGEN_VERSION_PATCH}
 ")
   install(FILES netgen_mesher-py3.egg-info DESTINATION ${NG_INSTALL_DIR_PYTHON} COMPONENT netgen)
 endif()
