diff -rupN --no-dereference netgen-6.2.2406/CMakeLists.txt netgen-6.2.2406-new/CMakeLists.txt
--- netgen-6.2.2406/CMakeLists.txt	2024-12-03 18:26:11.119472907 +0100
+++ netgen-6.2.2406-new/CMakeLists.txt	2024-12-03 18:26:11.383465621 +0100
@@ -296,7 +296,7 @@
        target_include_directories(netgen_gui INTERFACE ${TK_INCLUDE_PATH}/../xlib)
     endif()
 
-    target_link_libraries(nggui PUBLIC nglib togl PRIVATE "$<BUILD_INTERFACE:netgen_python>" )
+    target_link_libraries(nggui PUBLIC nglib togl jpeg avcodec avformat avutil swscale PRIVATE "$<BUILD_INTERFACE:netgen_python>" )
 
     if(WIN32)
       target_compile_definitions(netgen_gui INTERFACE -DTOGL_WGL)
@@ -417,9 +417,9 @@
       target_link_libraries(occ_libs INTERFACE Threads::Threads)
     endif()
     message(STATUS "OCC DIRS ${OpenCASCADE_INCLUDE_DIR}")
-    if(WIN32 AND USE_GUI)
-      target_link_libraries(nggui PRIVATE occ_libs Ws2_32.lib)
-    endif(WIN32 AND USE_GUI)
+    if(USE_GUI)
+      target_link_libraries(nggui PRIVATE occ_libs)
+    endif(USE_GUI)
 endif (USE_OCC)
 
 #######################################################################
