diff -rupN --no-dereference netgen-6.2.2406/libsrc/csg/vscsg.cpp netgen-6.2.2406-new/libsrc/csg/vscsg.cpp
--- netgen-6.2.2406/libsrc/csg/vscsg.cpp	2024-11-25 17:25:40.000000000 +0100
+++ netgen-6.2.2406-new/libsrc/csg/vscsg.cpp	2024-12-03 18:26:11.629458831 +0100
@@ -1,11 +1,11 @@
 #include <mystdlib.h>
-#include "incopengl.hpp"
 
 #include <myadt.hpp>
 #include <meshing.hpp>
 #include <csg.hpp>
 #include <stlgeom.hpp>
 
+#include "incopengl.hpp"
 #include <visual.hpp>
 
 #include "vscsg.hpp"
diff -rupN --no-dereference netgen-6.2.2406/libsrc/visualization/vsfieldlines.cpp netgen-6.2.2406-new/libsrc/visualization/vsfieldlines.cpp
--- netgen-6.2.2406/libsrc/visualization/vsfieldlines.cpp	2024-11-25 17:25:40.000000000 +0100
+++ netgen-6.2.2406-new/libsrc/visualization/vsfieldlines.cpp	2024-12-03 18:26:11.630458804 +0100
@@ -1,7 +1,6 @@
 #ifndef NOTCL
 
 #include <mystdlib.h>
-#include <incopengl.hpp>
 
 
 #include <myadt.hpp>
@@ -9,6 +8,7 @@
 #include <csg.hpp>
 #include <stlgeom.hpp>
 
+#include <incopengl.hpp>
 #include <visual.hpp>
 #include <meshing/fieldlines.hpp>
 
diff -rupN --no-dereference netgen-6.2.2406/ng/ngappinit.cpp netgen-6.2.2406-new/ng/ngappinit.cpp
--- netgen-6.2.2406/ng/ngappinit.cpp	2024-11-25 17:25:40.000000000 +0100
+++ netgen-6.2.2406-new/ng/ngappinit.cpp	2024-12-03 18:26:11.631458776 +0100
@@ -7,9 +7,9 @@
 #undef USE_TK_STUBS
 
 #include <mystdlib.h> 
+#include <core/mpi_wrapper.hpp>
 #include <inctcl.hpp>
 #include <meshing.hpp>
-#include <core/mpi_wrapper.hpp>
 
 #include "../libsrc/interface/writeuser.hpp"
 
