diff -rupN netgen-5.3.0/libsrc/meshing/parallelmesh.cpp netgen-5.3.0-new/libsrc/meshing/parallelmesh.cpp
--- netgen-5.3.0/libsrc/meshing/parallelmesh.cpp	2014-09-01 20:54:48.987249216 +0200
+++ netgen-5.3.0-new/libsrc/meshing/parallelmesh.cpp	2014-09-01 21:02:25.783230174 +0200
@@ -14,7 +14,7 @@ namespace metis {
 
 #if METIS_VER_MAJOR >= 5
 #define METIS5
-    typedef idx_t idxtype;   
+    typedef metis::idx_t idxtype;   
 #else
 #define METIS4
     typedef idxtype idx_t;  
@@ -794,10 +794,10 @@ namespace netgen
     eptr.Append (eind.Size());
     Array<idx_t> epart(ne), npart(nn);
 
-    int nparts = ntasks-1;
-    int edgecut;
+    metis::idx_t nparts = ntasks-1;
+    metis::idx_t edgecut;
 
-    int ncommon = 3;
+    metis::idx_t ncommon = 3;
     METIS_PartMeshDual (&ne, &nn, &eptr[0], &eind[0], NULL, NULL, &ncommon, &nparts,
 			NULL, NULL,
 			&edgecut, &epart[0], &npart[0]);
@@ -1029,11 +1029,11 @@ namespace netgen
     eptr.Append (eind.Size());
     Array<idx_t> epart(ne), npart(nn);
 
-    int nparts = ntasks-1;
-    int edgecut;
+    metis::idx_t nparts = ntasks-1;
+    metis::idx_t edgecut;
 
 
-    int ncommon = 3;
+    metis::idx_t ncommon = 3;
     METIS_PartMeshDual (&ne, &nn, &eptr[0], &eind[0], &nwgt[0], NULL, &ncommon, &nparts,
 			NULL, NULL,
 			&edgecut, &epart[0], &npart[0]);
@@ -1266,7 +1266,7 @@ namespace netgen
 
     for ( int vert = 0; vert < nn; vert++ )
       {
-	FlatArray<int> array ( cnt[vert], &adjacency[ xadj[vert] ] );
+	FlatArray<metis::idx_t> array ( cnt[vert], &adjacency[ xadj[vert] ] );
 	BubbleSort(array);
       }
 
@@ -1377,7 +1377,7 @@ namespace netgen
 
     for ( int el = 0; el < ne; el++ )
       {
-	FlatArray<int> array ( cnt[el], &adjacency[ xadj[el] ] );
+	FlatArray<metis::idx_t> array ( cnt[el], &adjacency[ xadj[el] ] );
 	BubbleSort(array);
       }
 
@@ -1428,8 +1428,8 @@ namespace netgen
   void Mesh :: PartDualHybridMesh2D ( ) 
   {
 #ifdef METIS
-    int ne = GetNSE();
-    int nv = GetNV();
+    metis::idx_t ne = GetNSE();
+    metis::idx_t nv = GetNV();
 
     Array<idxtype> xadj(ne+1);
     Array<idxtype> adjacency(ne*4);
@@ -1484,18 +1484,18 @@ namespace netgen
 
     idxtype *v_weights = NULL, *e_weights = NULL;
 
-    int weightflag = 0;
-    // int numflag = 0;
-    int nparts = ntasks - 1;
+    metis::idx_t weightflag = 0;
+    metis::idx_t numflag = 0;
+    metis::idx_t nparts = ntasks - 1;
 
-    int edgecut;
+    metis::idx_t edgecut;
     Array<idxtype> part(ne);
 
     for ( int el = 0; el < ne; el++ )
       BubbleSort (adjacency.Range (xadj[el], xadj[el+1]));
 
 #ifdef METIS4	
-    int options[5];
+    metis::idx_t options[5];
     options[0] = 0;
     METIS_PartGraphKway ( &ne, &xadj[0], &adjacency[0], v_weights, e_weights, &weightflag, 
 			  &numflag, &nparts, options, &edgecut, &part[0] );
