diff -rupN netgen-5.3.0/ng/ngpkg.cpp netgen-5.3.0-new/ng/ngpkg.cpp
--- netgen-5.3.0/ng/ngpkg.cpp	2014-09-01 20:54:49.423249198 +0200
+++ netgen-5.3.0-new/ng/ngpkg.cpp	2014-09-01 21:04:13.023225704 +0200
@@ -3136,13 +3136,11 @@ void PlayAnimFile(const char* name, int
   extern "C" int Ng_Init (Tcl_Interp * interp);
   extern "C" int Ng_CSG_Init (Tcl_Interp * interp);
 
-#ifdef _MSC_VER
   extern "C" int Ng_stl_Init (Tcl_Interp * interp);
   extern "C" int Ng_geom2d_Init (Tcl_Interp * interp);
   #ifdef OCCGEOMETRY
     extern "C" int Ng_occ_Init (Tcl_Interp * interp);
   #endif
-#endif
 
 
   // extern "C" int Ng_Geom2d_Init (Tcl_Interp * interp); 
@@ -3160,13 +3158,11 @@ void PlayAnimFile(const char* name, int
 #endif
 
     Ng_CSG_Init(interp);
-#ifdef _MSC_VER
     Ng_stl_Init(interp);
     Ng_geom2d_Init (interp);
 #ifdef OCCGEOMETRY
     Ng_occ_Init (interp);
 #endif
-#endif
 
 
     // Ng_Geom2d_Init(interp);
