--- netgen-6.2.2607/libsrc/core/utils.hpp.orig	2026-09-07 10:52:07.000000000 +0200
+++ netgen-6.2.2607/libsrc/core/utils.hpp	2026-09-23 17:20:08.130665873 +0200
@@ -16,7 +16,7 @@
 #include <mach/mach_time.h>
 #endif
 
-#ifdef NETGEN_ARCH_AMD64
+#if defined(NETGEN_ARCH_AMD64) || defined(__i686__)
 #ifdef WIN32
 #include <intrin.h>   // for __rdtsc()  CPU time step counter
 #else
@@ -63,7 +63,7 @@ namespace ngcore
 
   inline TTimePoint GetTimeCounter() noexcept
   {
-#if defined(NETGEN_ARCH_AMD64)
+#if defined(NETGEN_ARCH_AMD64) || defined(__i686__)
     return __rdtsc();
 #elif defined(NETGEN_ARCH_ARM64)
     unsigned long long tics;
--- netgen-6.2.2404/libsrc/core/python_ngcore.hpp.orig	2024-06-12 20:10:06.000000000 +0200
+++ netgen-6.2.2404/libsrc/core/python_ngcore.hpp	2026-09-19 21:23:45.953750296 +0200
@@ -182,10 +182,12 @@ namespace ngcore
     static std::string GetName() { return "D"; }
   };
 
+#ifdef __LP64__
   template<>
   struct PyNameTraits<size_t> {
     static std::string GetName() { return "S"; }
   };
+#endif
 
   template<typename T>
   struct PyNameTraits<std::shared_ptr<T>> {
